src/Datadog.Trace/TracingProcessManager.cs

```diff
@@ -146,25 +150,23 @@ private static void InitializePortManagerClaimFiles(string traceAgentDirectory)
                 Directory.CreateDirectory(portManagerDirectory);
             }
 
-            var currentProcess = Process.GetCurrentProcess();
-            _processName = currentProcess.ProcessName.ToLowerInvariant();
-            _processId = currentProcess.Id;
-            var fileClaim = Path.Combine(portManagerDirectory, _processId.ToString());
+            var fileClaim =
+                Path.Combine(
+                    portManagerDirectory,
+                    $"{DomainMetadata.ProcessId}_{DomainMetadata.AppDomainId}");
 
             var portManagerFiles = Directory.GetFiles(portManagerDirectory);
-            var deleted = 0;
             if (portManagerFiles.Length > 0)
             {
                 var activePids = Process.GetProcesses().Select(p => p.Id.ToString()).ToList();
```

Comment:
We don't want current culture settings messing with how pids are converted to strings (e.g. `1234`, `1,234`, `1.234`, even `1234.00`).

```suggestion
                var activePids = Process.GetProcesses().Select(p => p.Id.ToString(CultureInfo.InvariantCulture)).ToList();
```